src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -102,17 +103,19 @@ const ShareInviteDialog = (props: ShareInviteDialogProps): JSX.Element => {
     setUsersToInvite(newUserToInvite);
   };
 
-  const getUserPublicKey = async (email: string): Promise<string> => {
+  const getUserPublicKey = async (email: string): Promise<{ publicKey: string; publicKyberKey: string }> => {
     let publicKey = '';
+    let publicKyberKey = '';
     try {
       const publicKeyResponse = await userService.getPublicKeyByEmail(email);
       publicKey = publicKeyResponse.publicKey;
+      publicKyberKey = publicKeyResponse.publicKyberKey;
```

Comment:
the backend is already prepared to store this new key?

Reply:
Don't think so

Reply:
yes

Reply:
Is the backend ready?

Reply:
Still pending